apps/web/app/api/oauth/apps/[appId]/route.ts

```diff
@@ -46,6 +48,17 @@ export const PATCH = withWorkspace(
     } = updateOAuthAppSchema.parse(await parseRequestBody(req));
 
     try {
+      let logoUrl: string | undefined;
+
+      if (logo && !logo.startsWith(R2_URL)) {
+        const result = await storage.upload(
+          `integrations/${params.appId}_${nanoid(7)}`,
+          logo,
+        );
+
+        logoUrl = result.url;
+      }
```

Comment:
can you use the same image update flow that we use for workspace logo? specifically we need to delete the old logo if it's been uploaded before: https://github.com/dubinc/dub/blob/main/apps/web/app/api/workspaces/%5BidOrSlug%5D/logo/route.ts#L28-L37